source/documentation/infrastructure/getting-access.md

```diff
@@ -0,0 +1,113 @@
+# Accessing the Infrastructure
+
+## VPN
+
+All connections must be made via the GDS VPN. Please contact your local service desk for access.
+
+## Bastions
+
+The bastion servers act as a gateway to their respective clusters + databases.
+That is to say:
+
+- To access any Staging database or server, you must access via the staging bastion.
+- To access any Production database or server, you must access via the production bastion.
+
+### Accessing the Bastion
+
+#### Secrets
+
+[Extract the SSH Key secrets][getting-a-secret] for the bastion server:
+
+- staging secret name: `keys/govwifi-staging-bastion-key`
+- production secret name: `keys/govwifi-bastion-key`
+
+Find out the Elastic IPs for the bastion servers. You can do this by going into the AWS console,
+and find the instances with the Bastion name in.
+
+Remember that there are 2 regions, so there may be more than 2 bastions.
+
+#### SSH Config
+
+It is recommended to set up an ssh config for ease of use. All further instructions will
+assume you use similar naming.
+
+**Note**: The IP addresses have been redacted. Please substitute in the correct IP addresses.
+
+```
+AddKeysToAgent = yes
+
+Host govwifi-bastion-london-staging <redacted IP address>
+    Hostname <redacted IP address>
+    User ubuntu
+    IdentityFile ~/.ssh/govwifi/bastion-staging
+
+Host govwifi-bastion-london-production <redacted IP address>
+    Hostname <redacted IP address>
+    User ubuntu
+    IdentityFile ~/.ssh/govwifi/bastion-production
+
+Host govwifi-bastion-ireland-production <redacted IP address>
+    Hostname <redacted IP address>
+    User ubuntu
+    IdentityFile ~/.ssh/govwifi/bastion-production
+```
+
+You should now be able to connect to each of the hosts using ssh.
+
+```sh
+ssh govwifi-bastion-london-staging
+```
+
+## Databases
+
+There are 4 databases, all currently located in London. 2 for staging, 2 for production.
+
+To access each one, you will need to use their respective credentials and bastion server.
+
+### Admin database
```

Comment:
Can you add a one-liner to describe what this database is used for, or what information you can find there?